README.md

```diff
@@ -59,10 +59,10 @@ To install [@okta/okta-auth-js](https://www.npmjs.com/package/@okta/okta-auth-js
 
 ```bash
 # Run this command in your project root folder.
-npm install @okta/okta-auth-js --save
+yarn add @okta/okta-auth-js --save
 ```
 
-After running `npm install`, the minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
+After running `yarn add`, the minified auth client will be installed to `node_modules/@okta/okta-auth-js/dist`. You can copy the `dist` contents to a publicly hosted directory. However, if you're using a bundler like [Webpack](https://webpack.github.io/) or [Browserify](http://browserify.org/), you can simply import the module using CommonJS.
```

Comment:
We can probably make this more generic by saying:
"After running `yarn add`" ->  "After installing AuthJS"